acf-json/group_5ef954da477f5.json

```diff
@@ -28,30 +44,100 @@
             "ajax": 0,
             "placeholder": ""
         },
+        {
+            "key": "field_60527e8bb6c34",
+            "label": "Media Source",
+            "name": "media_source",
+            "type": "button_group",
+            "instructions": "",
+            "required": 0,
+            "conditional_logic": 0,
+            "wrapper": {
+                "width": "60",
+                "class": "",
+                "id": ""
+            },
+            "choices": {
+                "url": "URL",
+                "local": "Media Library"
+            },
+            "allow_null": 0,
+            "default_value": "local",
+            "layout": "horizontal",
+            "return_format": "value"
+        },
         {
             "key": "field_5ef9561897989",
             "label": "Media URL",
             "name": "hero_asset_url",
             "type": "url",
-            "instructions": "Enter the URL or file path to the Image or Video file for this Hero. If empty, Hero is not rendered.",
+            "instructions": "Enter a valid URL to the hero media",
             "required": 0,
             "conditional_logic": [
                 [
                     {
-                        "field": "field_60188dd996e59",
+                        "field": "field_60527e8bb6c34",
                         "operator": "==",
                         "value": "url"
                     }
                 ]
             ],
             "wrapper": {
-                "width": "60",
+                "width": "",
                 "class": "",
                 "id": ""
             },
             "default_value": "",
             "placeholder": ""
         },
+        {
```

Comment:
I don't think this Media File should be added in this PR. If we are going to switch, that would be in Zainab's PR that implements everything needed.

Reply:
Well, I already messed this one up. I didn't understand what you were saying, as the screen shot I got in the email notification of your comment looked like this, which was somewhat confusing:

<img width="846" alt="Screen Shot 2021-03-22 at 11 55 44 AM" src="https://user-images.githubusercontent.com/31013359/112043136-94a6f880-8b05-11eb-9431-e1c07a2c65c8.png">

And now I've already merged her branch, intending to go figure this one out later. Looking at the actual comment in context on Github makes more sense and now I see what you're saying.

Possibly ineffective counter-argument: my branch was about cleaning up field groups and associated blocks, where Zainab's was related primarily to swapping out a CSS background image for an actual `<img>` tag, so it's okay that my branch takes care of adding this field to the group and dealing with any conflicts the new field may cause.